test/src/org/exist/test/EmbeddedExistTester.java

```diff
@@ -1,140 +0,0 @@
-/*
```

Comment:
@dizzzz Sure, but he only used that because this PR has been languishing and not merged. As he is unwilling to do so, I will fix his PR.